openapi-cli/src/main/java/io/ballerina/openapi/cmd/CmdConstants.java

```diff
@@ -200,4 +200,9 @@ public String getValue() {
     // OS specific line separator
     public static final String LINE_SEPARATOR = System.lineSeparator();
     public static final String DOUBLE_LINE_SEPARATOR = LINE_SEPARATOR + LINE_SEPARATOR;
+    public static final String DO_NOT_MODIFY_LICENSE_HEADER = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n" +
+            "// This file is auto-generated by the Ballerina OpenAPI tool.\n\n";
+    public static final String DEFAULT_LICENSE_HEADER = "// AUTO-GENERATED FILE.\n" +
+            "// This file is auto-generated by the Ballerina OpenAPI tool.\n\n";
```

Comment:
These are not "license" headers right? IMO we can rename them as "file" headers instead

Reply:
apply where neccessary